<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChronologieAnomalie</class>
 <widget class="QWidget" name="ChronologieAnomalie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChronologieAnomalie</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelChronologie">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
     <property name="text">
      <string>Chronologie de l'inspection de </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="labelAnomalie">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
     <property name="text">
      <string>L'anomalie rencontrée</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidgetChronologie">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(238, 238, 236);
alternate-background-color:#606060;
selection-background-color:#282828;
</string>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTextBrowser" name="textBrowser">
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelImage1">
         <property name="text">
          <string>Image1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBoxImage" name="checkBox1">
         <property name="text">
          <string>Aucune image sélectionnée</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelImage_2">
         <property name="text">
          <string>Image2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBoxImage" name="checkBox_2">
         <property name="text">
          <string>Aucune image sélectionnée</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelImage_3">
         <property name="text">
          <string>image3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBoxImage" name="checkBox_3">
         <property name="text">
          <string>Aucune image sélectionnée</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pushButtonpdf">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Images.qrc">
       <normaloff>:/img/images/pdf-file.svg</normaloff>:/img/images/pdf-file.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>71</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="PushButtonAnnuler">
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCheckBoxImage</class>
   <extends>QCheckBox</extends>
   <header>qcheckboximage.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
